packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/content/WpfApplication-CSharp/MainWindow.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<Window x:Class="Company.WpfApplication1.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:Company.WpfApplication1"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="450" Width="800">
```

Comment:
did we want to change this to match what the default template is in net48?

Reply:
In a near-future change we'll probably want to do something like that. I think we'd want to spec what our project template ought to be like in consultation with @grubioe and @chabiss (Visual Studio) first.

Reply:
@vatsan-madhavan we need an item tracking this ASAP.  I would like the template to be changed for preview 5.

Reply:
@chabiss, if you want to change the template in winforms, we should discuss as well on this thread dotnet/winforms#663